tests/datasets/test_spacenet.py

```diff
@@ -263,7 +234,7 @@ def dataset(
 
     def test_getitem(self, dataset: SpaceNet5) -> None:
         # Iterate over all elements to maximize coverage
-        samples = [i for i in dataset]  # type: ignore[attr-defined]
+        samples = [dataset[i] for i in range(len(dataset))]
```

Comment:
I'm actually pretty confused why this used to work because `dataset` isn't iterable.